client/src/components/molecules/RequestsTable.tsx

```diff
@@ -0,0 +1,82 @@
+import { gql, useMutation } from '@apollo/client';
+import React, { FunctionComponent,  useState } from 'react';
+import  Form  from 'react-bootstrap/Form';
+import moment from 'moment';
+import Request from '../../data/types/request';
+import  Table  from 'react-bootstrap/Table';
+
+interface Props {
+    requests: Request[];
+}
+
+const RequestsTable: FunctionComponent<Props> = (props: Props) => {
+    const headingList = ['Fulfilled', 'Client Name', 'Quantity', 'Date Requested', '', ''];
+    const updateRequest = gql`
+    mutation updateRequest($request: RequestInput){
+        updateRequest(request: $request){
+          id
+          success
+          message
+        }
+      }
+    `;
+
+    const [requests, setRequests] = useState(props.requests);
+    const [mutateRequest] = useMutation(updateRequest);
+    console.log(requests);
+
+    const onFulfilledRequest = (index:number) => {
+        const requestsCopy = requests.slice();
+        const req = {...requestsCopy[index]};
+        req.fulfilled = !req.fulfilled;
+        requestsCopy[index]=req
+        setRequests(requestsCopy);
+        const id = req._id;
+        const requestId = req.requestId;
+        const fulfilled = req.fulfilled;
+        mutateRequest({variables:{request: {id, requestId, fulfilled}}});
+    }
+
+    
+    return (
+        <div className="request-list">
```

Comment:
When I run, I get this error:
```
index.js:1 Warning: validateDOMNesting(...): <div> cannot appear as a child of <tr>.
```

[According to StackOverflow](https://stackoverflow.com/questions/39915629/validatedomnesting-tr-cannot-appear-as-a-child-of-div), you might need to wrap `<tr>` in `<tbody>`